.github/workflows/spell-check-pr.yaml

```diff
@@ -0,0 +1,21 @@
+name: Check spelling
+
+on:
+  pull_request:
+
+jobs:
+  spellcheck:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v2
+
+      - name: Retrieve spell check dictionary
+        run: |
+          curl --silent --show-error \
+            --output .github/workflows/.cspell.json \
+            https://raw.githubusercontent.com/tier4/autoware-spell-check-dict/main/cspell/.cspell.json
```

Comment:
How about moving the autoware-spell-check-dict files to this repository?

Reply:
Hmm, it's possible, but I can't decide.
@kmiya What do you think? I'm concerned about CI scripts getting a bit complicated(to filter `.cspell.json`).